32/berubejd/selenium_testing.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3.8
+
+from pathlib import Path
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from time import sleep
+
+
+def setup(url=None):
+    if not url:
+        url = "http://pyplanet.herokuapp.com/"
+
+    print("\tStarting browser setup...")
+
+    chrome_driver = Path.cwd() / Path("chromedriver.exe")
```

Comment:
You might choose to make this cross platform (Path already handles the directory part!) by doing a check for Windows and tacking the .exe on the end if you're on that platform.

I feel like there should be something to do this in Pathlib already but I couldn't find it.

Reply:
This is a good point.  It is complicated slightly because I am running a Windows executable in Ubuntu under WSL.  Let me see if I can work something out.

Reply:
Applied a quick change that does a file search and, utilizing Path, should work on both Linux and Windows, including my WSL setup.